### PR TITLE
refactor: rappeling

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

- Refactors the rappeling logic
- Renames and fixes the helicoper model check
- Fixes some spotlight keybind settings
- Adds locales for spotlight and rappeling
- Adds the appropriate editorconfig

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -294,17 +299,22 @@ AddStateBagChangeHandler('spotlight', nil, function(bagName, _, value)
...
-        spotlight:disable(false)
+        if DoesVehicleHaveSearchlight(cache.vehicle) then
+            spotlight:disable(false)
+        end
+    elseif seat == 1 or seat == 2 then
+        spotlight:disable(true)
+
+        if DoesVehicleAllowRappel(cache.vehicle) then
+            rappel:disable(false)
+        end
```
Comment (Manason) [inline client/main.lua]: what about helicopters which fit more players? Should this be seat >= 1?

**Review by Manason — APPROVED** [review]

File: client/main.lua (hunk 1 above)
Comment (mafewtm) [reply client/main.lua]: I wrote this thinking that they should have to be on the door seats to be able to rappel out as it seemed weird for people to be able to do it from anywhere else. Do you think they should be able to do it anywhere besides the front two seats?

Comment (Manason) [reply client/main.lua]: Yeah I do as it's pretty cumbersome to force players to switch to the door seats to rappel, even though it makes sense.

Comment (mafewtm) [reply client/main.lua]: Sounds good. I'll push the change

**Pull request merged**